### PR TITLE
Solution for Single Number problem

### PR DESCRIPTION
**Pull request by miraliahmadli**

### CODE REVIEW

**Comment by sakomws** [comment]
Awesome thx @miraliahmadli ! Can you also, pls add detailed explanation as md file, and update the table in /leetcode/README.md: https://github.com/goupaz/algo/blob/master/starter/codes/leetcode/leetcode-136-sakows.md

**Comment by miraliahmadli** [comment]
> Awesome thx @miraliahmadli ! Can you also, pls add detailed explanation as md file, and update the table in /leetcode/README.md: https://github.com/goupaz/algo/blob/master/starter/codes/leetcode/leetcode-136-sakows.md

File: advanced/codes/go/leetcode-136-miraliahmadli.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+func singleNumber(nums []int) int {
+    var res = 0
+    for _, element := range nums{
+        res = res^element
+    }
+    return res
+}
```
Comment (sakomws) [inline advanced/codes/go/leetcode-136-miraliahmadli.go:7]: add new line pls

File: advanced/codes/python3/leetcode-136-miraliahmadli.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        res =0 
+        for elem in nums:
+            res= res^elem
+        return res
+    
+    # x^x = 0
+    # x^0 = x
+    # Order does not matter in XOR operation
+    # Since 'XOR'ing element with itself is zero, every element that appears twice will cancel each out and the remaining one wil be our answer
+    # Time Complexity = O(n), Space Complexity = O(1)
```
Comment (sakomws) [inline advanced/codes/python3/leetcode-136-miraliahmadli.py:12]: same here, new line add pls

**Review by sakomws — APPROVED** [review]
lgtm

**Pull request merged**